security-proxy/targets-mapping.properties

```diff
@@ -9,3 +9,5 @@ datafeeder=http://${DATAFEEDER_HOST}:8080/datafeeder/
 import=http://${IMPORT_HOST}:80/
 datahub=http://${DATAHUB_HOST}:80/datahub/
 ogc-api-records=http://${OGC_API_RECORDS_HOST}:8080/ogc-api-records/
+proxy=http://${SECURITY_PROXY_HOST}:8080/
+metadata-editor: http://${METADATA_EDITOR_HOST}:80/metadata-editor
```

Comment:
incorrect format no? should be `metadata-editor=`

And does the envs `METADATA_EDITOR_HOST` and `SECURITY_PROXY_HOST` exist? :thinking: I don't think so

Reply:
> They exists 😄

Not yet, it's a still a PR but ok I forgot to see this PR :).